plugins/dominant-color-images/hooks.php

```diff
@@ -36,6 +40,31 @@ function dominant_color_metadata( $metadata, $attachment_id ) {
 }
 add_filter( 'wp_generate_attachment_metadata', 'dominant_color_metadata', 10, 2 );
 
+/**
+ * Retrieves attachment metadata for attachment ID.
+ *
+ * This is a wrapper for {@see wp_get_attachment_metadata()} to add the typing which is augmented by the
+ * `wp_generate_attachment_metadata` via {@see dominant_color_metadata()}.
+ *
+ * @since n.e.x.t
+ *
+ * @param int $attachment_id Attachment post ID. Defaults to global $post.
```

Comment:
```suggestion
 * @param int $attachment_id Attachment post ID. Defaults 0.
```

Reply:
Actually, instead of adding a helper wrapper function with the type definition, maybe it would be better to add a stub function definition for PHPStan to read?

Reply:
Yes, a stub is better for consistent PHPDoc comments.

Reply:
I've implemented the stub in 3e8fed7 which improved a lot.